build.gradle

```diff
@@ -679,7 +680,6 @@ tasks.register('generateAssets') {
         if (usesMixins.toBoolean() && generateMixinConfig.toBoolean()) {
             def mixinConfigFile = getFile("src/main/resources/mixins.${modId}.json")
             if (!mixinConfigFile.exists()) {
-                def mixinConfigRefmap = "mixins.${modId}.refmap.json"
```

Comment:
Doesn't this also need to try and generate your refmap in the custom location, if it is specified and not already generated?

Reply:
the refmap is not generated before building the mod, this just changes where the default mixin config file will look for the refmap.
disabling `generateMixinConfig` still means any mixin config files must have the line `"refmap": "mixins.[modid].refmap.json"`.

this is the important part:
```diff
-mixinProviderSpec = modUtils.enableMixins(mixinProviderSpec, "mixins.${modId}.refmap.json")
+mixinProviderSpec = modUtils.enableMixins(mixinProviderSpec, mixinConfigRefmap)
```
the change on this line is just to make sure that if you *do* change the `mixinConfigRefmap` value before starting the game for the first time, it doesnt generate an invalid `mixinConfigFile` by pointing to a refmap that wont exist.